core/drivers/gic.c

```diff
@@ -679,9 +680,9 @@ static void gic_it_set_cpu_mask(struct gic_data *gd, size_t it,
 	target_shift = (it % NUM_TARGETS_PER_REG) * ITARGETSR_FIELD_BITS;
 	target &= ~(ITARGETSR_FIELD_MASK << target_shift);
 	target |= cpu_mask << target_shift;
-	DMSG("cpu_mask: writing 0x%x to 0x%" PRIxVA, target, itargetsr);
+	DMSG("cpu_mask: writing %#"PRIx32" to 0x%" PRIxVA, target, itargetsr);
```

Comment:
Nitpick, might as well use # instead of 0x for adresses, no?

Reply:
ok, i'll do.